etc/grafana/dashboards/reth-mempool.json

```diff
@@ -2324,7 +2324,7 @@
         "type": "prometheus",
         "uid": "${DS_PROMETHEUS}"
       },
-      "description": "Frequency of transaction types seen in announcements",
+      "description": "Duration of one call to fetch hashes pending fetch.\n\nFind Peer - find an idle fallback peer for a hash pending fetch.\n\nFill Request - fill `GetPooledTransactions` request, for the found peer, with more hashes from cache of hashes pending fetch. ",
```

Comment:
i'm not sure I follow what this means:

"Duration of one call to fetch hashes pending fetch"

can you rephrase?